### PR TITLE
[DO NOT MERGE] Update Mangement API rate limits

### PR DESCRIPTION
**Pull request by solepano**
Update documentation related to which Management API endpoints that are affected by rate limits.

### CODE REVIEW

File: articles/policies/rate-limits.md [hunk 1]
```diff
@@ -77,115 +77,31 @@ The following rate limits apply:
 - For __non-production tenants__ of enterprise customers, usage of the Management API is restricted to 2 requests per second (and bursts up to 10 requests).
 - For __paid__ tenants, usage of the Management API is restricted to 15 requests per second (and bursts up to 50 requests).
 
-The aforementioned rate limits include calls made via [Rules](/rules).
+The aforementioned rate limits include calls made via [Rules](/rules) and via the Manage Dashboard.
```
Comment (dafortune) [inline articles/policies/rate-limits.md:80]: Not sure if we want to clarify this, but limits associated with the dashboard are always the ones for prod tenants.

File: articles/policies/rate-limits.md [hunk 2]
```diff
@@ -77,115 +77,31 @@ The following rate limits apply:
...
-      <th><strong>POST</strong></th>
-      <th><strong>DELETE</strong></th>
-      <th><strong>PATCH</strong></th>
+      <th><strong>Method</strong></th>
+      <th><strong>Path</strong></th>
+      <th><strong>Affected Tenants</strong></th>
+      <th><strong>Limit</strong></th>
     </tr>
   </thead>
   <tbody>
```
Comment (dafortune) [inline articles/policies/rate-limits.md:98]: What about `/oidc/register`?

**Comment by pmalouin** [comment]
@solepano 
Should this PR be completed to consider the api2 rate-limits changes complete?

**Comment by rkhoriander** [comment]
@solepano, we are working on cleaning up some old Docs PRs. Can you please provide an update on this PR?

**Pull request closed without merging**